<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1555327431604' stop='1555327431671'>
    <name>adding a new contact with name, email, and phone number</name>
    <title>adding a new contact with name, email, and phone number</title>
    <test-cases>
        <test-case start='1555327431619' status='failed' stop='1555327431644'>
            <name>should send an email address</name>
            <title>should send an email address</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: chrome not reachable
                  (Session info: chrome=72.0.3626.109)
                  (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 4.15.0-46-generic x86_64)</message>
                <stack-trace>WebDriverError: chrome not reachable
                  (Session info: chrome=72.0.3626.109)
                  (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 4.15.0-46-generic x86_64)
                    at Object.checkLegacyResponse (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebDriver.navigate().to(data:text/html,&lt;html>&lt;/html>)
                    at Driver.schedule (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at Navigation.to (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:1133:25)
                    at Driver.get (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:988:28)
                    at driver.controlFlow.execute.then.then (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/built/browser.js:675:32)
                    at ManagedPromise.invokeCallback_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
                    at /home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: Run beforeAll in control flow
                    at UserContext.&lt;anonymous> (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at queueRunnerFactory (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at UserContext.fn (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5325:13)
                    at attempt (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                    at /home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
                From asynchronous test: 
                Error
                    at Suite.describe (/home/jihene/Documents/GithubPFE/RunRepository-/e2e/add-second-contact.e2e-spec.ts:4:3)
                    at addSpecsToSuite (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/home/jihene/Documents/GithubPFE/RunRepository-/e2e/add-second-contact.e2e-spec.ts:3:1)
                    at Module._compile (internal/modules/cjs/loader.js:689:30)
                    at Module.m._compile (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/ts-node/src/index.ts:299:25)
                    at Module._extensions..js (internal/modules/cjs/loader.js:700:10)
                    at Object.require.extensions.(anonymous function) [as .ts] (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/ts-node/src/index.ts:302:14)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1555327431644' status='failed' stop='1555327431671'>
            <name>should send a phone number</name>
            <title>should send a phone number</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: chrome not reachable
                  (Session info: chrome=72.0.3626.109)
                  (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 4.15.0-46-generic x86_64)</message>
                <stack-trace>WebDriverError: chrome not reachable
                  (Session info: chrome=72.0.3626.109)
                  (Driver info: chromedriver=2.46.628388 (4a34a70827ac54148e092aafb70504c4ea7ae926),platform=Linux 4.15.0-46-generic x86_64)
                    at Object.checkLegacyResponse (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebDriver.navigate().to(data:text/html,&lt;html>&lt;/html>)
                    at Driver.schedule (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at Navigation.to (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:1133:25)
                    at Driver.get (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:988:28)
                    at driver.controlFlow.execute.then.then (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/built/browser.js:675:32)
                    at ManagedPromise.invokeCallback_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
                    at /home/jihene/Documents/GithubPFE/RunRepository-/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: Run beforeAll in control flow
                    at UserContext.&lt;anonymous> (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at queueRunnerFactory (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at UserContext.fn (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5325:13)
                    at attempt (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                    at /home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
                From asynchronous test: 
                Error
                    at Suite.describe (/home/jihene/Documents/GithubPFE/RunRepository-/e2e/add-second-contact.e2e-spec.ts:4:3)
                    at addSpecsToSuite (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/home/jihene/Documents/GithubPFE/RunRepository-/e2e/add-second-contact.e2e-spec.ts:3:1)
                    at Module._compile (internal/modules/cjs/loader.js:689:30)
                    at Module.m._compile (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/ts-node/src/index.ts:299:25)
                    at Module._extensions..js (internal/modules/cjs/loader.js:700:10)
                    at Object.require.extensions.(anonymous function) [as .ts] (/home/jihene/Documents/GithubPFE/RunRepository-/node_modules/ts-node/src/index.ts:302:14)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>